translations/messages.fr.xlf

```diff
@@ -1370,6 +1374,66 @@
+      </trans-unit>
+      <trans-unit id="2932513123180634807" datatype="html">
+        <source>Multiple laps sowing</source>
+        <target>Distribution à plusieur tours</target>
+      </trans-unit>
+      <trans-unit id="1218183221112518220" datatype="html">
+        <source>However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, do such a move!</source>
+        <target>Cependant, la distribution ne s'arrête que quand la dernière graine est déposée dans une maison qui contient (avant déposage) zéro ou trois graines.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites un mouvement comme celà !</target>
+      </trans-unit>
+      <trans-unit id="4651814494564877992" datatype="html">
+        <source>So, after landing in the house with 3 seeds, a second lap has been started</source>
+        <target>Donc, après atterissage dans la maison avec 3 graines, un second tour a commencé</target>
+      </trans-unit>
+      <trans-unit id="3067825694318214397" datatype="html">
+        <source>Captures during distribution (1/2)</source>
+        <target>Captures pendant la distribution (1/2)</target>
+      </trans-unit>
+      <trans-unit id="7526762213389215712" datatype="html">
+        <source>If, during some distribution, you pass by one of your house that contains 3 seeds, and drop a fourth seed, you capture the house immediately, then continue the distribution!&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, do such a move!</source>
+        <target>Si, durant une distribution, vous passez pas une de vos maison qui contient 3 graines, et déposez une quatrième graine, vous capturez la maison immédiatement, puis continuez la distribution !&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites un tel mouvement !</target>
+      </trans-unit>
+      <trans-unit id="9198664076697170304" datatype="html">
+        <source>Congratulations, you captured 4 seeds.</source>
+        <target>Félicitations, vous capturez 4 graines.</target>
+      </trans-unit>
+      <trans-unit id="8936894363705168622" datatype="html">
+        <source>Captures during distribution (2/2)</source>
+        <target>Captures pendant la distribution (2/2)</target>
+      </trans-unit>
+      <trans-unit id="4075686340012399389" datatype="html">
+        <source>If, during some distribution, you pass by one house of the opponent that contains 3 seeds, and drop a fourth seed, the opponent capture the house immediately (while you continue to distribute)!&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, do such a move!</source>
+        <target>Si, durant une distribution, vous passez pas une maison de l'adversaire qui contient 3 graines, and déposez une quatrième, l'adversaire capture la maison immédiatement (pendant que vous continuez la distribution) !&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites un tel mouvement !</target>
```

Comment:
and -> et

Reply:
and implique et ? Quelle sornette logisticienne raconte tu là ?